operational/flexera/msp/parent_child_group_sync/README.md

```diff
@@ -0,0 +1,58 @@
+# MSP Parent to Child Group Sync
+
+## What It Does
+
+This policy template synchronizes groups, roles and users from a parent organization to child MSP organizations based on group description patterns. Groups in the parent org with descriptions matching the pattern `{GroupSync Prefix}_{OrgID}_{Group Name}` will be synchronized to the specified child org.  Using the default Group Sync Prefix, an example configuration string looks like:
```

Comment:
Extra space here: "specified child org.  Using the"

Might be worth keeping the "What It Does" section fairly short and adding a "How It Works" section to cover the details, such as the pattern.